gitbootcamp.rst

```diff
@@ -551,3 +551,35 @@ To edit an open pull request that targets ``main``:
       git push git@github.com:<contributor>/cpython <pr_number>:<branch_name>
 
 5. Optionally, :ref:`delete the PR branch <deleting_branches>`.
+
+
+GitHub CLI
+----------
+
+GitHub CLI is a CLI (command-line interface) to work with GitHub.
```

Comment:
Actually, we don't need to repeat "CLI":

```suggestion
GitHub CLI is a command-line interface to work with GitHub.
```

Reply:
Addressed